### ISSUE TITLE
Error: Custom element doesn't exist: weather-card-chart

### ISSUE THREAD

**Issue by KattenJansson1974** [Yevgenium/weather-chart-card#32]
What am I doing wrong? I get this in my UI

```
Custom element doesn't exist: weather-card-chart.

type: 'custom:weather-card-chart'
title: Weather
weather: weather.openweathermap
```

Regardless what I do, it still pops up. I'm certainly missing something here... please if anyone can assist.

**Comment by KattenJansson1974** [comment]
Ok, almost there... or not. According to the instructions, the .js-file is called "custom-weather-card-chart.js" but the file itself is just called "weather-card-chart.js".

So no error-message, but... a completely white page, also my other cards are not visible.

**Comment by mihail4anov** [comment]
a completely white page, also my other cards are not visible.

The same problem

**Comment by mihail4anov** [comment]
Fixed
Do this https://www.home-assistant.io/integrations/openweathermap/

**Comment by too-far-away** [comment]
I'm having the same issue with `HA 0.118.2`.
The following installation steps were performed:
1. Copied the `weather-card-chart.js` to `/usr/share/hassio/homeassistant/www/`
2. In `Configuration->Lovelace Dashboards->Resources` added a new resource with URL set to `/local/weather-card-chart.js` and the resource type `JavaScript Module`
3. Added custom card with `type: 'custom:weather-card-chart'`. In the process it already shows the error listed in the subject, but eventually allows saving the new card.
Then the card looks like below:
![image](https://user-images.githubusercontent.com/60689789/100345322-d3f78980-2fea-11eb-8291-ce03a8088900.png)
What am I doing wrong?

**Comment by darazi** [comment]
Sorry for beating a dead horse. I can't for the life of me get this integrated. 
So far I copied the weather-card-chart.js to /usr/share/hassio/homeassistant/www/  (and restarted) 
I then tried calling on the card in Lovelace and got the "doesn't exist" error. Even after flushing the cache (I'm on Chrome). 

I am not able to get a clear understanding of what declaration lines are needed in configuration.yaml. 

Any ideas? 

Edit:

Got it working! Added the git hub reposetory to HCAS.  Thank you @Yevgenium! Really awesome card.
